packages/renderer/src/lib/pod/DeployPodToKube.spec.ts

```diff
@@ -325,3 +342,36 @@ test('Fail to deploy ingress if service is not selected', async () => {
   // Expect kubernetesCreateIngress to not be called since we error out as service wasn't selected
   await waitFor(() => expect(kubernetesCreateIngressMock).not.toHaveBeenCalled());
 });
+
+test('Should display Open pod button after successful deployment', async () => {
+  kubernetesGetCurrentContextNameMock.mockResolvedValue('default');
+  await waitRender({});
+  const createButton = screen.getByRole('button', { name: 'Deploy' });
+  expect(createButton).toBeInTheDocument();
+  expect(createButton).toBeEnabled();
+
+  kubernetesCreatePodMock.mockResolvedValue({
+    metadata: { name: 'hello', namespace: 'default' },
+  });
+  kubernetesReadNamespacedPodMock.mockResolvedValue({
+    metadata: { name: 'hello' },
+    status: {
+      phase: 'Running',
+    },
+  });
+
+  vi.useFakeTimers();
```

Comment:
Please use the `waitfor(() => expect(kubernetesCreateMock....` that is used in other tests for better uniformity. See `When deploying a pod, restricted security context is added` test few line above

Reply:
Thanks! Wrapped mock calls with `waitfor` call.